### ISSUE TITLE
@import vs. @require

### ISSUE THREAD

**Issue by schneikai** [metaskills/less-rails#51]
Usually you have just one stylesheet in your Rails project. But sometimes there are certain pages in the project that have a very complex design and it would be a waste to load those styles on every other page. So you make the layout load application.css and a controller specific stylesheet. Now you want to use twitter-bootstrap-rails in your project so you put _@import "twitter/bootstrap/bootstrap";_ in your application.css.less. But you also want to use some variables and mixins from Bootstrap in your controller specific stylesheets too. You would need to _@import "twitter/bootstrap/bootstrap";_ which would add all the styles from Bootstrap again but you already have them in your application.css.

Long story short: What abou a _@require_ statement that, in contrast to _@import_, would just load the referenced file for compilation but would not add any content to the compiled file? Does it make sense?
